roles/matrix-bridge-appservice-slack/defaults/main.yml

```diff
@@ -33,7 +33,7 @@ matrix_appservice_slack_slack_port: 9003
 matrix_appservice_slack_container_http_host_bind_port: ''
 
 matrix_appservice_slack_homeserver_media_url: "{{ matrix_server_fqn_matrix }}"
-matrix_appservice_slack_homeserver_url: "http://matrix-synapse:8008"
+matrix_appservice_slack_homeserver_url: "http://matrix-synapse:{{ matrix_synapse_container_client_api_port }}"
```

Comment:
We should move this to `group_vars/matrix_servers`. You can define `matrix_appservice_slack_homeserver_url` as empty here and ensure it's set via `roles/matrix-bridge-appservice-slack/tasks/validate_config.yml `